### DISCUSSION TITLE
한번에 끝내는 컨테이너 기반

### DISCUSSION THREAD

**Discussion by hyungkishin** [tedilabs/fastcampus-devops#79] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/55230332/182025888-5640cd14-a4de-4bff-8f77-5c79d9ccb05b.png)

뭔가 Terraform 에 대한 부연설명은 따로 공부하고 와야되는것 같네요 ... 

이전 어딘가 아니면 다른 강의에서 하신부분이 끼워져있는 느낌이고, 해당 질문이 많은것으로 보아 그냥 섹션에 끼워넣는게 필요할것 같은데 

아니면 어떻게 해결해야 하는지 확인 부탁드릴 수 있을까요 .. ?

**Comment by hyungkishin** [comment, marked as answer]
위 에러는 해결이 됬습니다 ㅠㅠㅠ 
다만 
network 설정을 마치고 ec2 설정을 하는 중에
![image](https://user-images.githubusercontent.com/55230332/182028141-3d79a698-a4cf-4928-a383-b2cca4331524.png)
4분 까지 갔다가 
![image](https://user-images.githubusercontent.com/55230332/182028277-a715390d-b1ab-4388-b251-e31e54b492a9.png)
뻗어버리는것 같습니다

**Reply by hyungkishin** [reply]
<img width="476" alt="image" src="https://user-images.githubusercontent.com/55230332/182030330-aa5f410f-0d86-47c1-82a8-97dfeed7fe1c.png">

ssh 를 강제로 등록해주니 잘 되는것 같습니다 ... :) 

그리고 강사님 강의 중에 초격자 가 하나 더 있더라구여 ... :)
구매 해서 terraform 상세히 보는중입니다

**Reply by hyungkishin** [reply]
약간 눈물 겹습니다 

<img width="659" alt="image" src="https://user-images.githubusercontent.com/55230332/182030418-68ed4142-7bed-4f2b-8ec3-e1221a3888cf.png">

**Reply by posquit0** [reply]
안녕하세요! 제가 요새 개인적인 사정으로 확인이 너무 늦었네요. 죄송합니다.

- 도커 & 쿠버네티스 강좌는 제가 강사진으로 참여한 강의는 아니오나, 패스트캠퍼스 측에서 데브옵스 초격차 패키지의 도커 파트를 일부 기초 섹션에 활용하고 싶다고 요청을 받았습니다. 그러다보니 컨텐츠 자체가 데브옵스 초격차에 엮여있는 것들이 많아 혼란을 초래하고 있는 것 같습니다. 😭 
- 사실 도커 컨테이너를 실습하기 위한 리눅스 환경이 필요한 것이라, 해당 실습 코드를 꼭 성공시킬 필요는 없습니다. 가상머신이나 남는 PC 장비에 우분투를 설치하고 docker, docker-compose 등을 설치하여 앞으로 강의를 진행하셔도 괜찮습니다.
- 초격차 강의까지 구매하시면서 성공하셨다니 감동이네요... ㅎㅎ DevOps에 관심이 있으시다면, Packer / Terraform / Ansible / Docker / Kubernetes 기초를 익히시는데 도움이 되었으면 합니다. ㅎㅎ
